### PR TITLE
MODDATAIMP-52 Create Profile entity

### PR DESCRIPTION
**Pull request by Igor-Gorchakov**
Created Profile entity
Created JobExecutionProfile entity

### CODE REVIEW

File: examples/mod-source-record-manager/jobExecutionProfile.sample [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (OleksiiKuzminov) [inline examples/mod-source-record-manager/jobExecutionProfile.sample]: maybe just "jobProfile" ?

File: examples/mod-source-record-manager/profile.sample [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "1kl9m60r-fvb7-45n0-23c0-13n8gbkl7091",
+  "name": "Profile for bib files",
+  "state": "COMMITTED"
```
Comment (OleksiiKuzminov) [inline examples/mod-source-record-manager/profile.sample]: we need state?

Comment (Igor-Gorchakov) [reply examples/mod-source-record-manager/profile.sample]: yes, we need state for the Profile but not for JobExecutionProfile.

File: examples/mod-source-record-manager/jobExecutionProfile.sample (hunk 1 above)
Comment (Igor-Gorchakov) [reply examples/mod-source-record-manager/jobExecutionProfile.sample]: agree

**Pull request merged**